config.toml

```diff
@@ -14,7 +14,10 @@ author = "Dany Marcoux"
 description = "Blog of Dany Marcoux"
 keywords = "blog,web development,developer"
 info = "Full Stack Web Developer"
-avatarPath = "images/me.jpg" # Under static/
+# Images are under the static directory.
+# Hugo will use this image as the website thumbnail whenever sharing a blog post
+images = ["images/me.jpg"]
```

Comment:
I don't know why, but somehow the meta tag `og:image` from [_internal/opengraph.html](https://gohugo.io/templates/internal/#open-graph) is ignoring the `images/` part in the `images` array and trying to load `me.jpg` directly... This is why I duplicated `static/images/me.jpg` to `static/me.jpg`. The thumbnail needs a different ratio anyway, so perhaps it's the way to go.

Test this with https://socialsharepreview.com/?url=https://deploy-preview-19--blog-dmarcoux-com.netlify.app/